src/librustdoc/core.rs

```diff
@@ -492,6 +492,9 @@ crate fn run_global_ctxt(
             tcx.ensure().check_mod_attrs(module);
         }
     });
+    tcx.sess.time("rustc_check_passes", || {
```

Comment:
I wouldn't expect this to take very long - can you run `-Z time-passes` locally and see if it takes more than, say, 10 milliseconds? If not I don't think it's worth timing.
```suggestion
    tcx.sess.time("check_unused_or_stable_features", || {
```

Reply:
I'll simply remove it then.